pyproject.toml

```diff
@@ -50,8 +50,8 @@ pythonpath = [
 [tool.black]
 exclude = "tests/fixtures/*"
 
-[tool.setuptools]
-packages = ["torchfix"]
+[tool.setuptools.packages.find]
+include = ["torchfix*"]
```

Comment:
Could you explain why is this needed?

Reply:
It's not needed, I removed it. 

(In my local code base another directory with feature development was causing `setuptools` to discover multiple top-level packages, but that's unrelated to this PR)